### PR TITLE
Adding Ocean

### PR DESCRIPTION
**Pull request by JakubDynak**
This pull request was made by the Gatsby Branch Editor.

### CODE REVIEW

**Comment by Andre601** [comment]
He even added some weird extra - on the lists...

https://github.com/Terminal/Discord_Fork/pull/145/commits/15e1e57ff02149f2cfc2eef0d6bf0296c6eed6fb#diff-88143c8736320c6722af98885e28ab1bR20

**Comment by lepon01** [comment]
image

**Comment by lepon01** [comment]
Is this command necessary?

image

**Comment by lepon01** [comment]
My favorite number

image

**Comment by lepon01** [comment]
Check for correct input before continuing

image

File: data/en/bots/null.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+avatar: >-
+  https://cdn.discordapp.com/avatars/437737030567329805/0903dca19b4e772b26463f33fcaefb12.png?size=2048
+cover: null
+description: Invite This Amazing Bot
```
Comment (lepon01) [inline data/en/bots/null.md]: Is this a good description?

**Pull request merged**